### ISSUE TITLE
libssl.so.1.1: cannot open shared object file

### ISSUE THREAD

**Issue by lmiq** [tectonic-typesetting/tectonic#1025]
The binary that is download by default in Linux does not work in modern versions of Ubuntu (or Mint), because the a library is missing:

```
./tectonic 
./tectonic: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory

```

It is not trivial how to install such a library, unfortunately.

**Comment by vlasakm** [comment]
I am unfortunately not able to help you with the issue, but I can at least link to https://github.com/tectonic-typesetting/tectonic/issues/892, which seems to describe the same issue.

**Comment by trimclain** [comment]
This fix worked for me:
https://stackoverflow.com/a/72633324

**Comment by cdelledonne** [comment]
On Fedora 38, I had to install the package `openssl1.1`

**Comment by pkgw** [comment]
As of the next release, the default builds will cut over to building against OpenSSL3 (#1092), which should make this problem go away.

**Comment by pkgw** [comment]
Yes, sorry, I just let this slide. But I've just put out 0.15.0 with this change (among others):

https://github.com/tectonic-typesetting/tectonic/releases/tag/tectonic%400.15.0

**Comment by rm-dr** [comment]
Should be fixed with 0.15.0
